cmd2/cmd2.py

```diff
@@ -4281,11 +4281,11 @@ def _validate_cmdfinalization_callable(cls, func: Callable[[plugin.CommandFinali
         signature = inspect.signature(func)
         _, param = list(signature.parameters.items())[0]
         if param.annotation != plugin.CommandFinalizationData:
-            raise TypeError("{} must have one parameter declared with type "
-                            "'cmd2.plugin.CommandFinalizationData'".format(func.__name__))
+            raise TypeError("{} must have one parameter declared with type {}".format(func.__name__,
+                                                                                      plugin.CommandFinalizationData))
         if signature.return_annotation != plugin.CommandFinalizationData:
```

Comment:
Should we be comparing both `param.annotation` and `signature.return_annotation` to `plugin.CommandFinalizationData`?